edx_django_utils/hooks/exceptions.py

```diff
@@ -0,0 +1,16 @@
+"""
+Exceptions thrown by Hooks.
+"""
+
+
+class HookException(Exception):
+    """
+    Base exception for hooks. It is re-raised by the Pipeline Runner if any of
+    the actions/filters that is executing raises it.
+    """
+
+    def __init__(self, message='', redirect_to=None, status_code=None):
```

Comment:
I think that we need to look for all the filters that we have planed to use to identify which other parameters could be useful here.

Reply:
I'll work on that. Do you have any suggestions @felipemontoya ?

Reply:
Do we need to put the list here? can we use kwargs instead and be 100% flexible to add more params later?

Reply:
Yes! Good idea. I already added kwargs